tools/windows-node-installer/test/e2e/aws_test.go

```diff
@@ -71,18 +69,19 @@ func testDestroyWindowsInstance(t *testing.T) {
 	t.Run("test installer json file is deleted", testInstallerJsonFileIsDeleted)
 }
 
+// awsSetup does all the prerequisite steps untill the instance creation part.
```

Comment:
untill -> until

Reply:
prerequisite steps needed for what? This comment doesn't tell me anything that the function does. Please add more detail